package.json

```diff
@@ -36,7 +36,6 @@
   "homepage": "https://github.com/mojaloop/central-services-error-handling#readme",
```

Comment:
Please update version to 7.4.3 and run npm install

Reply:
Version updated to 7.4.3 and npm install has been ran